components/atom/tooltip/src/index.scss

```
@@ -8,6 +8,7 @@ $maw-atom-tooltip: 250px !default;
 $mah-atom-tooltip: 100px !default;
 $c-atom-tooltip: $c-white !default;
 $bgc-atom-tooltip: $c-black !default;
+$bdc-atom-tooltip: $bgc-atom-tooltip !default;
```

I don't really like the idea of coupling a border colour variable to a background colour variable... couldn't you just do the following? 
```suggestion
$bdc-atom-tooltip: $c-black !default;
```